tracee-rules/engine/stats.go

```diff
@@ -0,0 +1,11 @@
+package engine
+
+import "github.com/aquasecurity/tracee/tracee-rules/stats"
+
+func (e *Engine) Stats() stats.RulesStats {
```

Comment:
They're in the same package, I wanted to separate them because I personally find it more readable that way (engine.go contains only functions relevant to it's own "real" function and this is more of a side export). I can move it back if that convention is important.

Reply:
TBH I have never seen methods with the same receiver split across multiple Go source files. Then if you look at my other comments I think it's easier to inject metrics to the Engine as optional arg at construction time.

Reply:
I went for an approach where the engine exposes it's stat struct pointer. To me splitting receiver functions seems fine but i'm not going to argue with the project's existing conventions, so I merged these functions back (also helps that they are smaller now)